README.md

```diff
@@ -1,15 +1,49 @@
-# song-stitch
+# Song Stitch
 
 <p align="center">
   <img alt="SongStitch Logo" src="https://raw.githubusercontent.com/SongStitch/song-stitch/main/public/logo.png" width="500px"/>
 </p>
 
 <div align="center">
 
-[![Website Status](https://img.shields.io/website?style=flat-square&up_message=UP&url=https%3A%2F%2Fsongstitch.art%2F
-)](https://songstitch.art/)
+[![Website Status](https://img.shields.io/website?style=flat-square&up_message=UP&url=https%3A%2F%2Fsongstitch.art%2F)](https://songstitch.art/)
 [![CI status](https://img.shields.io/github/actions/workflow/status/SongStitch/song-stitch/deploy.yml?branch=main&style=flat-square)](https://github.com/SongStitch/song-stitch/actions?query=branch%3Amain)
 [![License](https://img.shields.io/github/license/SongStitch/song-stitch?style=flat-square)](/LICENCE)
 
 </div>
 
+<div align="center">
+A <em>blazingly fast</em> web application for generating LastFM collages, written in Go.
+</div>
+
+<br/>
+
+<p align="center">
+  <img alt="SongStitch Logo" src="https://raw.githubusercontent.com/SongStitch/song-stitch/main/collage.png" width="300px"/>
+</p>
+
+### Requirements
+
+- Docker
```

Comment:
These aren't strict requirements! Should add golang, but not sure on which version.

hmm might be better than adding `go mod` docs etc. though